src/app/public/modules/select-field/select-field.component.html

```diff
@@ -1,4 +1,7 @@
-<div class="sky-select-field">
+<div
+  class="sky-select-field"
+  (focusout)="onTouched()"
```

Comment:
We have three "touch" methods on the component: `_onTouched`, `onTouched`, and `touch`. It's getting a tad confusing. Think we could rename this to `onHostFocusOut` (or something similarly descriptive)?

Reply:
I can't rename the public onTouched method since that would be breaking, but I can move the touch methods contents into it and make a new public method called onHostFocusOut. I'll also rename the _onTouched to be registeredOnTouchCallback to be explicit about what it is